pyre/check.py

```
@@ -4000,15 +4007,52 @@ def failed_bound(measured, baseline_value):
             bound = failed_bound(median_elapsed, median_baseline)
             return (
                 bound is None, bound, median_elapsed, median_baseline,
-                f"median {PERF_RETRY_RUNS}",
+                self._retry_sample_note(
+                    backend, measured_times, baseline_key, baseline_times,
+                ),
             )
 
         return False, bound, elapsed, baseline_time, ""
 
+    def _retry_sample_note(
```

Ruff (0.16.3)

[warning] Missing return type annotation for private function `_retry_sample_note`

(ANN202)